recipes/short-read-connector/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+# decrease RAM needed
+sed -i.bak 's/make -j/make -j1/' INSTALL
+sed -i.bak 's/2> log_linker_err//' test/simple_test.sh
+
+# change run_discoSnp++.sh deps path
+sed -i.bak 's/\$EDIR\/bin/\$EDIR/' short_read_connector.sh
```

Comment:
@cmonjeau you can also write this in the following way: `'s|\$EDIR/bin|\$EDIR|'`

Reply:
Done